mkdocs.yml

```diff
@@ -10,34 +11,34 @@ edit_uri: ""
 copyright: Copyright &copy; 2018 - 2024 TrenchBoot Developers
 
 nav:
-    - Home: index.md
-    - Events:
+  - Home: index.md
```

Comment:
First indent level is 2 spaces, and all of the following ones are 4, is this what lint suggested?

Reply:
I think it depends on config: https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.indentation

Currently from what I can see it wants 2 spaces between first character in word (that is not including `-`) in current line and first character (including `-`) in next line. In examples below I used dots for those 2 spaces:


*  ```yaml
    text:
    ..- text2
    ```

*  ```yaml
    - text:
      ..- text2
    ```

*  ```yaml
    text:
    ..text2
    ```

Reply:
Interesting, definitely not what I expected, and difficult for text editors without knowledge of YAML syntax. Since this seems to be the standard way of formatting YAML files, I'm marking it as resolved, even though I still don't like it :slightly_smiling_face: